scripts/batch_preprocess_known_faces.py

```
@@ -0,0 +1,31 @@
+import os
+import cv2
+import numpy as np
+from src.core.face_engine_haar_lbph import load_and_correct_orientation, apply_clahe, FACE_SIZE
+
+KNOWN_FACES_DIR = "data/known_faces"
+
+def preprocess_and_save(face_img_path, save_path):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)